src/StreamJsonRpc/JsonMessageFormatter.cs

```diff
@@ -1515,26 +1491,17 @@ public override void WriteJson(JsonWriter writer, object? value, JsonSerializer
                 {
                     writer.WriteNull();
                 }
+                else if (this.interfaceType.IsAssignableFrom(value.GetType()) is false)
```

Comment:
Will change to `!`.
Myself, I never liked the `!` syntax and even preferred `== false` to it because the `!` at the beginning of the statement is sometimes hard to see. Also `is false` resembles natural language more.
But will adapt to the local codestyle and use `!` here :-)